src/main/scala/com/microsoft/partnercatalyst/fortis/spark/TransformContext.scala

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{BlacklistedTerm, SiteSettings}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.image.ImageAnalyzer
+import com.microsoft.partnercatalyst.fortis.spark.transforms.language.LanguageDetector
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.LocationsExtractorFactory
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.SentimentDetectorAuth
+import org.apache.spark.broadcast.Broadcast
+
+case class TransformContext(
+  siteSettings: SiteSettings = null,
+  langToWatchlist: Broadcast[Map[String, List[String]]] = null,
+  blacklist: Broadcast[List[BlacklistedTerm]] = null,
+  locationsExtractorFactory: Broadcast[LocationsExtractorFactory] = null,
+  imageAnalyzer: ImageAnalyzer = null,
```

Comment:
Would it be worth a divider comment to denote the following fields as "cheap to create so not broadcastet" to make it obvious why only some of the fields in this object are broadcasts?